### ISSUE TITLE
Image gets rotated when plugin is applied

### ISSUE THREAD

**Issue by amongmany** [MengTo/Angle-Sketch-Plugin#51]
Sometimes when I use the plugin, the image from the selected artboard is applied to the angle shape with a 90° rotation. This is rather frustrating.

**Comment by WilNichols** [comment]
Getting this regularly, and "rotate mockup" does nothing. Makes the plugin largely unusable.

**Comment by WilNichols** [comment]
My current workaround is as follows.

1. Export the desired screen contents @10x
2. Create a new new artboard, rotated in the same way that Angle incorrectly rotates the mockup
3. Add the newly-exported raster to the artboard, and resize to fit. This often means distorting the raster from portrait to landscape, but by scaling down by 10x, it's not pixellated.
4. Reapply mockup with Angle, using the new distorted asset .

**Comment by F-DK** [comment]
This is still an issue — now in Sketch 69.2 and 70.1 — and @WilNichols' suggestion doesn't work. It seems to be an issue with some of the mockups, e.g. all '05 iPhone 8/06/*' mockups. But by checking the source Sketch files I couldn't find what's causing the problem.

Since I needed these for the our App Store previews this

**Comment by WilNichols** [comment]
@F-DK You are correct, that doesn't work reliably. I've found that if I copy the target device artboard and the screen artboard into a blank document, run the plugin, and then use the rotate command, it actually works. You can then copy the new-rotated content back.

**Comment by F-DK** [comment]
What I found out worked for me was fixing the issue at the source. As it seems the starting point of the 'screen' layer ins the mockup is at the wrong corner. By rotating the screen layer and then adjusting the layer's shape to fit exactly as before the issue can was fixed.

This also means the author is likely aware of this issue but seems to have different priorities. Since this is a simple fix and merely labor to be done it's a pity everyone has to build their own crooked workarounds or fixes at the source — but cannot share them with others. Lots of redundant work. And this isn't open source but a paid product.

@MengTo I'm fully aware that dealing with thousands of github issues in various repos is tough but there are a lot of open issues a lot of designers doing the author's work, everyone on their own. That's neither efficient nor effective.

**Comment by F-DK** [comment]
> … run the plugin, and then use the rotate command …
![image](https://user-images.githubusercontent.com/1479114/100726496-4c809080-33c5-11eb-8575-ea25ef0e65ed.png)

@WilNichols Do you mean these commands from the Angle plugin?
Either I'm using them wrong or they don't work for me. I never experienced them having any effect — and couldn't find any sort of documentation.

Whenever I use them I get the hint "Please select a shape, Angle mockup or Angle composition" — which is what I did.
